src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -75,9 +76,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
     String content = CharStreams.toString(request.getReader());
-    // TODO: Get actual video timestamp from request.
-    // Use the start of the video for now.
+
+    // Default timestamp is the start of the video.
     long timestampMilliseconds = 0;
+    if (request.getParameter(PARAM_TIMESTAMP) != null) {
```

Comment:
In the case where this condition is true, will the comment timestamp be the start of the video?

Reply:
Yep, we default to the start of the video if a timestamp isn't provided.  If a timestamp isn't provided, this comment is a reply to another comment and we use that timestamp when ordering on the page.

It's possible nether are provided, in which case I should probably send an error...

Reply:
Do we want the comment to be at the start or should we send an error for that as well?

Reply:
I will cleanup the reply vs root comment doPost logic in a future PR

Reply:
tracking issue #164